NAMESPACE

```diff
@@ -1,3 +1,4 @@
 export(reliabilityUniDimFrequentist)
 export(IntraclassCorrelation)
 export(reliabilityUniDimBayesian)
+export(cohenFleissKappa)
```

Comment:
```suggestion
export(cohensFleissKappa)
```
I think this would be more consistent since everything in the analysis is named that way